ICAFIX/ReApplyFixPipeline.sh

```diff
@@ -395,8 +395,8 @@ main()
 	# are set in ${FSL_FIXDIR}/settings.sh, which is sourced below for interpreted modes.
 	# Note that the ciftiopen.m, ciftisave.m functions are *appended* to the path through the ${FSL_FIX_CIFTIRW} 
 	# environment variable within fix_3_clean.m itself.
-	# Note that we are NOT adding '${HCPPIPEDIR}/global/matlab' to the path (which also contains the
-	# CIFTI I/O functions as well), so the versions in ${FSL_FIX_CIFTIRW} will be the ones that get used
+	# Note that we are NOT adding '${HCPPIPEDIR}/global/matlab' to the path (which used to contain the
+	# CIFTI I/O functions as well), or HCPCIFTIRWDIR, so the versions in ${FSL_FIX_CIFTIRW} will be the ones that get used
```

Comment:
Should we consider changing this behavior at this time?  i.e., Make the CIFTI I/O in this script use what is specified by `${HCPCIFTIRWDIR}` rather than `${FSL_FIX_CIFTIRW}` (for interpreted mode)?

Reply:
I think it would be reasonable to override anything in the FIX settings.